CONTRIBUTING.md

```diff
@@ -22,40 +22,40 @@ ideal report includes:
 First, clone the repo, then `cd` into the repo.
 
 ```bash
-$ git clone git@github.com:vortechsa/python-sdk.git
-$ cd python-sdk
+git clone git@github.com:vortechsa/python-sdk.git
+cd python-sdk
 ```
 
 create a new virtual environment
 
 ```bash
-$ python3.7 -m venv venv
+python -m venv venv
```

Comment:
nit: probably we should mention that when creating the virtual env they should use 3.11?